From d149637cd659bd5874ce57f3ae1a607deee16b5b Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Thu, 15 Dec 2016 06:54:45 +0800
Subject: [PATCH 1354/5242] MLK-13615-1 regulator: pf1550-regulator-rpmsg:
 should set voltage closest to min_uV

commit  70cba2a5c1cd8a5c73b0a163868886f3d93f4b76 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to API definition, Set the voltage for the regulator within the
range specified, the driver should select the voltage closest to min_uV.

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/regulator/pf1550-regulator-rpmsg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/pf1550-regulator-rpmsg.c b/drivers/regulator/pf1550-regulator-rpmsg.c
index 51515282..0bebc7e 100644
--- a/drivers/regulator/pf1550-regulator-rpmsg.c
+++ b/drivers/regulator/pf1550-regulator-rpmsg.c
@@ -168,7 +168,7 @@ static int pf1550_set_voltage(struct regulator_dev *reg,
 
 	msg.cmd = PF1550_SET_VOL;
 	msg.regulator = reg->desc->id;
-	msg.voltage = uV;
+	msg.voltage = minuV;
 
 	err = pf1550_send_message(&msg, info);
 	if (err)
-- 
1.7.9.5

